src/qt/overviewpage.cpp

```diff
@@ -157,6 +157,9 @@ OverviewPage::OverviewPage(const PlatformStyle *platformStyle, QWidget *parent)
 
     // start with displaying the "out of sync" warnings
     showOutOfSyncWarning(true);
+    connect(ui->labelWalletStatus, SIGNAL(clicked()), this, SLOT(handleOutOfSyncWarningClicks()));
+    connect(ui->labelTransactionsStatus, SIGNAL(clicked()), this, SLOT(handleOutOfSyncWarningClicks()));
```

Comment:
Two labels above are `QLabel`s in Dash and are not clickable, corresponding connects should be removed.
```diff
diff --git a/src/qt/overviewpage.cpp b/src/qt/overviewpage.cpp
index dd6bd6b27..08472a17c 100644
--- a/src/qt/overviewpage.cpp
+++ b/src/qt/overviewpage.cpp
@@ -157,9 +157,6 @@ OverviewPage::OverviewPage(const PlatformStyle *platformStyle, QWidget *parent)
 
     // start with displaying the "out of sync" warnings
     showOutOfSyncWarning(true);
-    connect(ui->labelWalletStatus, SIGNAL(clicked()), this, SLOT(handleOutOfSyncWarningClicks()));
-    connect(ui->labelTransactionsStatus, SIGNAL(clicked()), this, SLOT(handleOutOfSyncWarningClicks()));
-
 
     // that's it for litemode
     if(fLiteMode) return;
```
Some other related `connect`s on overview screen are also useless due to this fact I guess, at least until this
>The most important ones are 2 commits that allow to bring up the overlay by clicking on the spinning icon or the progress label.

 is implemented.

Reply:
I tried to figure out what happened here. It looks like in bitcoin these are actually buttons. This change was reverted while merging in 0.12 changes. The PR is https://github.com/dashpay/dash/pull/698, the commit is https://github.com/dashpay/dash/commit/ad5da28cd86188eef3916958482b2271f8509d18
Do you remember why you had to revert those?

Reply:
Yep, we didn't want that warning icon all over the place back then and I doubt we want to change this now too. IMO clicking on the spinning icon or the progress bar/label would make more sense (I would maybe even go with spinning icon only).

Reply:
You say "change this now too". Do you mean not change it forever or just not now? In case you want to change it someday in the future, I'd suggest to stick with the connect calls as they will then come in handy at that time.

I really like the clicking on the progress bar. When I've first seen the overlay in bitcoin and clicked on hide, it was kind of intuitional that I had to click on the progress bar to get it back. At that time I didn't even notice the small spinning icon on the bottom right. Also, the overlay moves smoothly to the bottom, so it's kind of clear that you have to bring it back up by clicking somewhere near where it vanished.

Reply:
I mean "nothing changed, should be no icons here". Can't predict the future ;) but if we ever would think about changing this to buttons for some reason I think we will add required `connect`s at that time. Currently they are just spawning errors in debug.log (and make no sense).